src/test/java/com/wikia/webdriver/common/dataprovider/MercuryCategoriesDataProvider.java

```diff
@@ -0,0 +1,17 @@
+package com.wikia.webdriver.common.dataprovider;
+
+import org.testng.annotations.DataProvider;
+
+public class MercuryCategoriesDataProvider {
+
+    @DataProvider
+    public static final Object[][] getCategoriesWithTitles() {
+        return new Object[][]{
+                {"/wiki/Category:NoContent", "NOCONTENT"},
+                {"/wiki/Category:WithContent", "WITHCONTENT"},
+                {"/wiki/Category:DisplayTitleWithContent", "CUSTOM DISPLAY TITLE WITH CONTENT"}
+                //Below case can be enabled when XW-1461 is fixed
+                //{"/wiki/Category:DisplayTitleNoContent", "CUSTOM DISPLAY TITLE WITHOUT CONTENT"}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)